miio/airpurifier_miot.py

```diff
@@ -134,102 +195,76 @@ def mode(self) -> OperationMode:
     @property
     def buzzer(self) -> Optional[bool]:
         """Return True if buzzer is on."""
-        if self.data["buzzer"] is not None:
-            return self.data["buzzer"]
+        buzzer = self.data.get("buzzer", None)
+        if buzzer is not None:
+            return buzzer
 
         return None
 
     @property
     def child_lock(self) -> bool:
         """Return True if child lock is on."""
-        return self.data["child_lock"]
+        return self.data.get("child_lock", None)
 
     @property
     def filter_life_remaining(self) -> int:
         """Time until the filter should be changed."""
-        return self.data["filter_life_remaining"]
+        return self.data.get("filter_life_remaining", None)
```

Comment:
All "remaining" properties should return `timedelta()`s, but considering this has been already like this it's not a big deal, but this API will likely get consolidated at latest when purifiers get a common interface.